Umweltzone/src/main/res/values-cs-rCZ/strings.xml

```diff
@@ -0,0 +1,98 @@
+    <string name="city_info_geometry_updated_at_text">"Poslední aktualizace oblasti: <xliff:g example="Sep 6, 2014" id="updatedAt">%1$s</xliff:g>     "</string>
+    <string name="city_info_allowed_exhaust_emission_standard_in_diesel_prohibition_zone">Do oblastí zakazujících diesely mohou vjet vozidla, která splňují alespoň emisní normu Euro <xliff:g example="6" id="number">%1$d</xliff:g>.</string>
+    <string name="city_info_zone_number_for_residents_since">"Pro obyvatele toto nařízení platí od <xliff:g example="May 31, 2018" id="date">%1$s</xliff:g>.    "</string>
+    <string name="city_info_zone_number_for_non_residents_since">"Toto nařízení platí pro nerezidenty od <xliff:g example="May 31, 2018" id="date">%1$s</xliff:g>.     "</string>
+    <string name="city_info_abroad_licensed_vehicle_zone_info">Vozidla registrovaná v zahraničí mohou vjet do nízkoemisní zóny do  <xliff:g example="Jan 31, 2016" id="date">%1$s</xliff:g>  <xliff:g example="with a yellow or a green" id="with_colors">%2$s</xliff:g> plaketou</string>
+    <string name="appinfo_license_content">Zdrojový kód aplikace Umweltzone je licencován dle GPLv3 Licence. Zeměpisné souřadnice získané z OpenStreetMap jsou licencovány na základě Open Database Licence.</string>
+    <string name="appinfo_description_content">Tato aplikace zobrazuje polohu nízkoemisních zón (LEZ) a oblastí se zákazem dieselů (DPZ), a obsahuje další fakta týkající se těchto témat. Vjezd do LEZ je omezen na vozidla s konkrétní emisní plaketou. Plaketa  odpovídá emisní úrovni vozidla. Vozy, které nesplňují určitou emisní normu dieselových motorů nemohou vjet do DPZ. Cílem těchto omezení je zlepšení městské kvality vzduchu. Jemné částice musí být redukovány z důvodu zachování &#8217;zdraví &#8212; obyvatel. Kromě mapy a seznamu měst tato aplikace obsahuje podrobnou obrazovku s jednotlivými informacemi pro vybrané město. V menu (&#8220;FAQ&#8221;) lze nalézt často kladené otázky. Krom toho Umweltzone obsahuje odkazy na stránky s dalšími informacemi o LEZ, DPZ a dalších podobných tématech.</string>
+    <string name="appinfo_references_libraries_content">Aplikace využívá těchto softwarových knihoven.</string>
+    <string name="zone_not_drawable_notice">"   <![CDATA[&#8230; aby<b> nízkoemisní zóna %1$s</b> mohla být zobrazena. Problém je, že %2$s nedodává zeměpisné souřadnice (hodnoty zeměpisné šírky a délky) popisující polohu nízkoemisní zóny v strojově čitelném formátu. Potřebné informace nelze získat z obrázku, pdf souboru či webové stránky. Z tohoto důvodu není možné zobrazit přesnou polohu a tvar nízkoemisní zóny. <br />         <br />  Tuto situaci můžete změnit skrz <b> e-mail </b> Obsah e-mailu je předpřipraven, jen přidejte své jméno na konec a odešlete. ]]>     "</string>
+    <string name="appinfo_references_texts_content">"Je-li text převzat odjinud, je vždy ozdrojován. "</string>
```

Comment:
Please remove the outer double quotes and the whitespace. They cause formatting issues. It might be that the editor added them without your knowledge.